i18n/id.json

```diff
@@ -37,7 +37,9 @@
   "I18N_FOOTER_BROWSE_LIBRARY": "Lihat Perpustakaan",
   "I18N_FOOTER_CONTACT_ALL_CAPS": "HUBUNGI KAMI",
   "I18N_FOOTER_CONTACT_US": "Hubungi Kami",
+  "I18N_FOOTER_CONTRIBUTE_ALL_CAPS": "SUMBANG",
```

Comment:
Thanks -- we can change that! How about the other two?

Note that this one is "CONTRIBUTE", which is a header for three other items: Get Involved, Donate, and Credits. See https://github.com/oppia/oppia/pull/2200/files/4e4ebcef36fa07e800169728fd688d45113fb3f3#diff-f3d6b06ab7d2290964aa390abb2d0e57R26